ci/test/04_install.sh

```diff
@@ -108,6 +108,13 @@ export DIR_FUZZ_IN=${DIR_QA_ASSETS}/fuzz_seed_corpus/
 
 DOCKER_EXEC mkdir -p "${BASE_SCRATCH_DIR}/sanitizer-output/"
 
+if [[ ${USE_MEMORY_SANITIZER} == "true" ]]; then
+  DOCKER_EXEC mkdir -p "${BASE_SCRATCH_DIR}/msan/build/"
+  DOCKER_EXEC git clone https://github.com/llvm/llvm-project "${BASE_SCRATCH_DIR}/msan/llvm-project"
```

Comment:
Just testing locally and it takes a long time to download all of this, mostly it times out. Would it be possible to set `--depth=1`?

Reply:
Oh, of course! Thanks! Now updated.